<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BV4_STATEX_Student</class>
 <widget class="qMRMLWidget" name="BV4_STATEX_Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>509</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="inputBox_Student_Name"/>
   </item>
   <item>
    <widget class="QLineEdit" name="inputBox_Exam_Number">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Load_Structures">
     <property name="text">
      <string>Ladda in strukturer</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_1">
     <property name="text">
      <string>Struktur 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_2">
     <property name="text">
      <string>Struktur 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_3">
     <property name="text">
      <string>Struktur 3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_4">
     <property name="text">
      <string>Struktur 4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_5">
     <property name="text">
      <string>Struktur 5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_6">
     <property name="text">
      <string>Struktur 6</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_7">
     <property name="text">
      <string>Struktur 7</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_8">
     <property name="text">
      <string>Struktur 8</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_9">
     <property name="text">
      <string>Struktur 9</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Structure_10">
     <property name="text">
      <string>Struktur 10</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
